### PR TITLE
FH-4055 Add Keycloak integration details

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by aidenkeating** [comment]
@maleck13 @philbrookes Would one of you mind taking a look?

File: ui/public/views/integration-docs/mobile-app-integration.html [hunk 1]
```diff
@@ -36,70 +36,118 @@
...
+    npm install --save keycloak-js@1.0.4
+  </pre>
+
+  <pre ng-if="clientType == 'cordova' && service.name == 'keycloak'" class="prettyprint lang-bash indented-code">
+      npm install --save fh-mobile-core
+      npm install --save keycloak-js@1.0.4
+  </pre>
+
+  <h3>Getting started</h3>
+  <pre ng-if="clientType == 'cordova' && service.name == 'fh-sync-server' && ! enabled('keycloak',service)" class="prettyprint lang-js">
```
Comment (philbrookes) [inline ui/public/views/integration-docs/mobile-app-integration.html]: if you add the class `indented-code` to this `<pre>` tag, then you can indent the code in the source HTML without it affecting the output. Which keeps the HTML a bit easier to scan.

Comment (aidenkeating) [reply ui/public/views/integration-docs/mobile-app-integration.html]: @philbrookes Adding in the `indented-code` class to `pre` just seems to flatten the whole block of code. This is the output I see when using it:

```
const mobileCore = require('fh-mobile-core');
const sync = require('fh-sync-js');
const mcpConfig = require('../mcpConfig.json');

mobileCore.configure(mcpConfig).then((config) => {
const syncConfig = config.getConfigFor('fh-sync-server');
sync.init({
cloudUrl: syncConfig.uri,
storage_strategy: 'dom'
});
sync.manage('myDataset', null, {}, {}, () => {
// Initialise the rest of your app.
});
});
```
My CSS is terrible at best so I'm not sure why this is happening from a first look. I'll see if I can fix it

Comment (philbrookes) [reply ui/public/views/integration-docs/mobile-app-integration.html]: oh yes, you're right that would happen, you're original code is best then!

**Pull request merged**